src/main/kotlin/com/faforever/userservice/controller/OAuthController.kt

```diff
@@ -104,13 +109,15 @@ class OAuthController(
                                 .build()
                                 .toUriString()
                         )
+                        is LoginResult.TechnicalError -> redirectTechnicalError(response)
                     }
                 }
         }
 
     @GetMapping("/consent")
     fun showConsent(
         request: ServerHttpRequest,
+        response: ServerHttpResponse,
```

Comment:
Was this added just for completeness it doesn't seem used

Reply:
Yeah there was a lot of stuff to clean up.